main.tf

```
@@ -165,6 +165,7 @@ resource "random_id" "cloud_build_notifier_service" {
   # We use a keeper here so we can force cloud run to redeploy on script change.
   keepers = {
     script_hash = google_storage_bucket_object.cloud_build_notifier_config.md5hash
+    slack_template_hash = google_storage_bucket_object.slack_template.md5hash
```

Need to format this so the linter will pass:
```suggestion
    script_hash         = google_storage_bucket_object.cloud_build_notifier_config.md5hash
    slack_template_hash = google_storage_bucket_object.slack_template.md5hash
```